### PR TITLE
:bug: Cannot send if balance not more than 0

### LINKED ISSUE #155: 🐛 Fix: Do not allow sending when 0 balance is available
# ℹ Overview

image

### 💻 System Configuration

- Operating System:
    - Edition: 
    - Version: 
    - Build: 
- Browser:
    - Type: 
    - Version:

### PR DESCRIPTION
**Pull request by treasurechic**
# ℹ Overview

### 📝 Related Issues

-- closes #155

### CODE REVIEW

File: components/SendAssetForm.js [hunk 1]
```diff
@@ -155,17 +156,18 @@ const SendAssetForm = ({
...
               disabled={
-                formattedAddresses.length < 1 ||
-                !wallet ||
-                !assetId ||
-                !csvTransactions
+                !assetBalance.success &&
+                (!wallet ||
+                  !assetId ||
+                  !csvTransactions ||
+                  0 > parseInt(assetBalance.message))
```
Comment (atrefonas) [inline components/SendAssetForm.js]: The 0 > parseInt(assetBalance.message) is kind of weird. How can 0 be greater than it? Is the .message a negative value? Please explain.

**Review by atrefonas — CHANGES_REQUESTED** [review]

**Comment by atrefonas** [comment]
The balance is 0 and I am still able to send?

image

**Comment by treasurechic** [comment]
I made a mistake with the expressions earlier, fixed now

**Comment by atrefonas** [comment]
image

The Send Assets button is still enabled here even with 0 available. Can you please confirm on your end the fix before requesting review?

**Comment by treasurechic** [comment]
I confirmed this and it worked, are you using the newly generated link: https://algodex-mailbox-c8ljchidw-algodex-dev.vercel.app/en/send-assets

**Comment by atrefonas** [comment]
Yes it is the right link

image

**Comment by treasurechic** [comment]
Unable to reproduce it as It works here using the same link


Screen Shot 2022-04-08 at 11 54 20 PM
.

**Comment by treasurechic** [comment]
Here is a screenshot of another test using a grok4 asset

Screen Shot 2022-04-09 at 12 06 58 AM

**Pull request merged**